packages/flutter/test/material/text_form_field_test.dart

```diff
@@ -431,6 +431,28 @@ void main() {
     expect(find.text('changedValue'), findsOneWidget);
   });
 
+  testWidgets('onChanged callbacks value and FormFieldState.value are sync', (WidgetTester tester) async {
+    FormFieldState<String> state;
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: Center(
+            child: TextFormField(
+              onChanged: (String value) {
+                expect(value, state.value);
```

Comment:
Can you flip a boolean value here to mark that the onChanged callback was called, then add an expectation after the `enterText` call to verify that the callback was called?

Reply:
Good point, thank you, now updated.